apps/src/code-studio/components/progress/ScriptOverviewHeader.jsx

```diff
@@ -88,16 +126,41 @@ class ScriptOverviewHeader extends Component {
             width={SCRIPT_OVERVIEW_WIDTH}
           />
         }
+        {showScriptVersionWarning &&
+          <Notification
+            type={NotificationType.warning}
+            notice={i18n.wrongCourseVersionWarningNotice()}
+            details={i18n.wrongCourseVersionWarningDetails()}
+            dismissible={true}
+            width={SCRIPT_OVERVIEW_WIDTH}
+          />
+        }
```

Comment:
The only difference between this notification and the one immediately above is the `details` prop.  It might be DRYer to vary that one prop; unless we sometimes need to show both of these at the same time?

Reply:
Good suggestion! updated. Nope, we never need to show both, which is enforced on the server.